### PR TITLE
chore: swap event placeholders

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

I haven't done much change

- linking chapter's events

---

`/dashboard/chapter/{id}/`

image

### CODE REVIEW

**Comment by gikf** [comment]
Ideally this should be split into two PRs. One for the events listing and one for venue form changes.

File: client/src/modules/dashboard/Venues/components/VenueForm.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const fields: Fields[] = [
     key: 'street_address',
     label: 'Street address',
     placeholder: 'Street address',
-    isRequired: true,
+    isRequired: false,
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/components/VenueForm.tsx:51]:
```suggestion
    isRequired: true,

File: client/src/modules/dashboard/Venues/components/VenueForm.tsx [hunk 2]
```diff
@@ -155,7 +155,7 @@ const VenueForm: React.FC<VenueFormProps> = (props) => {
         onSubmit={handleSubmit(onSubmit)}
         className={styles.form}
       >
-        <VStack>
+        <VStack gap={'1em'}>
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/components/VenueForm.tsx:158]:
```suggestion
        <VStack>

File: client/src/styles/Form.module.css [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .form {
     display: flex;
+    margin-block: 2em;
```
Comment (Sboonny) [inline client/src/styles/Form.module.css:3]:
```suggestion
```

**Review by Sboonny — COMMENTED** [review]
revert venue form

File: client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx [hunk 4]
```diff
@@ -108,7 +117,49 @@ export const ChapterPage: NextPage = () => {
         </ProgressCardContent>
       </Card>
       <Text fontWeight={400} margin={2}>
```
Comment (gikf) [inline client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:119]:
```suggestion
```
This likely isn't going to be needed.

File: client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx [hunk 5]
```diff
@@ -108,7 +117,49 @@ export const ChapterPage: NextPage = () => {
...
+                    >
+                      Invite only
+                    </Tag>
+                  )}
+                </Flex>
+              </Flex>
+            </GridItem>
+          ))}
+        </Grid>
       </Text>
```
Comment (gikf) [inline client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:163]:
```suggestion
```

File: client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx [hunk 6]
```diff
@@ -108,7 +117,49 @@ export const ChapterPage: NextPage = () => {
         </ProgressCardContent>
       </Card>
       <Text fontWeight={400} margin={2}>
-        PlaceHolder for Events...
+        <Heading as="h2" marginBlock={3}>
```
Comment (Sboonny) [inline client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:120]:
```suggestion
        <Heading as="h2" marginBlock={3} fontSize={'lg'}>

File: client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx [hunk 7]
```diff
@@ -108,7 +117,49 @@ export const ChapterPage: NextPage = () => {
...
+                      maxH={'2em'}
+                    >
+                      Invite only
+                    </Tag>
+                  )}
+                </Flex>
+              </Flex>
+            </GridItem>
+          ))}
+        </Grid>
```
Comment (gikf) [inline client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:162]:
```suggestion
      <Heading as="h2" marginBlock={3} fontSize={'lg'}>
        Organized Events
      </Heading>
      <Grid gap={'2em'}>
        {data.chapter.events.map(({ name, canceled, invite_only, id }) => (
          <GridItem key={id}>
            <Flex justifyContent={'space-between'}>
              <LinkButton href={`/events/${id}`}>{name}</LinkButton>
              <Flex>
                {canceled && (
                  <Tag
                    borderRadius="lg"
                    marginRight={'3'}
                    paddingInline="[1 , 2]"
                    paddingBlock="[.5, 1]"
                    fontSize={['small', 'md']}
                    maxWidth={'8em'}
                    mt="1"
                    maxH={'2em'}
                    colorScheme={'red'}
                  >
                    Canceled
                  </Tag>
                )}
                {invite_only && (
                  <Tag
                    borderRadius="lg"
                    mt="1"
                    paddingInline="[1 , 2]"
                    paddingBlock="[.5, 1]"
                    colorScheme={'blue'}
                    fontSize={['small', 'md']}
                    maxWidth={'8em'}
                    maxH={'2em'}
                  >
                    Invite only
                  </Tag>
                )}
              </Flex>
            </Flex>
          </GridItem>
        ))}
      </Grid>
```
Just dedent to keep linter happy.

Comment (Sboonny) [reply client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:162]: Thank you 👍 💚

File: client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx [hunk 8]
```diff
@@ -1,11 +1,20 @@
-import { Heading, Link, Box, HStack, Text } from '@chakra-ui/layout';
+import {
+  Heading,
+  Link,
+  Box,
+  HStack,
+  Text,
```
Comment (Sboonny) [inline client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx:6]:
```suggestion
```

**Review by gikf — APPROVED** [review]
LGTM :+1: 

It looks pretty nice too.

**Review by ojeytonwilliams — APPROVED** [review]
This does look nice. Thanks @Sboonny

**Pull request merged**